echo/integrations/svelte-email.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: "Svelte E-mail"
+---
+
+Integrating Novu Echo with [Svelte email](https://react.email/) for your Svelte application can be done in three steps. If you don't have an app, you can [clone our Svelte example](https://github.com/novuhq/novu-svelte-email).
+
+<Steps>
+  <Step title="Install Svelte email components">
+    Install the required Svelte email components.
```

Comment:
Is it one component or many?

Reply:
Many components in one package. @sumitsaurabh927

Reply:
@unicodeveloper 
Oh, okay. 
So is it correct to write 'email components' or shall we write 'email component'?

Reply:
It's correct to write 'email components'. That's how we wrote it for the others. Another alternative is to just use "package" but I want it to be consistent with the other integration pages. @sumitsaurabh927